### ISSUE TITLE
Documentation changes

### ISSUE THREAD

**Issue by ngreenwald** [vanvalenlab/kiosk-console#298]
Google Cloud Setup Section: 
- [x] Step 2: Kubernetes. Needs more documentation. Steps 1: User has to enable APIs. Step 2: Search for kubernetes, click enable.  Step 3: Add credentials to your project. Select "Yes" to question 1. After that I got the following message: "You don't need to create new credentials.
For your situation you can use Application Default Credentials, which provide a simple way to access Google APIs from App Engine or Compute Engine. Learn how to use Application Default Credentials. If you prefer, you can create a service account: https://developers.google.com/identity/protocols/application-default-credentials?hl=en_US
https://console.cloud.google.com/iam-admin/serviceaccounts?project=deepcellsoftware&folder&organizationId
I got super lost at this point, not sure how to proceed. Luckily our main project already has kubernetes enabled, so I went ahead on that one. 

- [x] Step 4: GPU quotas. I don't have an option for GPUs (all regions). I have to set quotas for individual GPUs. e.g., our GCP instance has a quota of 4 NVIDIA V100 GPUs specifically. I think more explicit instructions here would be helpful. "From the 'Metrics' dropdown menu, select NVIDIA V100 GPU and request at least 1." 

- [x] Step 4: IP address quotas. Same thing for IP Addresses: "From the 'Metrics' dropdown menu, scroll to 'In-use IP Addresses' and make sure the limit is at least 16. 

- [x] Step 5: Create a bucket. I would specify that the user should select Standard storage class and fine-grained access control. I don't think the second one matters, but the less guessing during installation the better. 

Starting the Kiosk:

- [x] For the note that starts "This command and the one that follows may need..."  it could be useful to also include a comment about needing to supply a password 

Deecell Kiosk Usage:

- [x] "select the configuration option for your chosen cloud provider" It isn't obvious that Configure GKE is the actual command I need to select; I'd put this in the description as well. 

- [x]  "Name your cluster." Because the project and bucket are pre-specified names, it's not obvious that the cluster is being created here, rather than referencing something else that already exists like for the other two options

- [x] Bug with firewalls / account upgrade

One fix for firewall issue is pushed I'll continue!

**Comment by willgraf** [comment]
Thanks so much for these great suggestions! I have a couple of clarifying questions below:

> Step 4: GPU quotas. I don't have an option for GPUs (all regions). I have to set quotas for individual GPUs. e.g., our GCP instance has a quota of 4 NVIDIA V100 GPUs specifically. I think more explicit instructions here would be helpful. "From the 'Metrics' dropdown menu, select NVIDIA V100 GPU and request at least 1."
> Step 4: IP address quotas. Same thing for IP Addresses: "From the 'Metrics' dropdown menu, scroll to 'In-use IP Addresses' and make sure the limit is at least 16.

I am hesitant to give direct instructions on using the GCP website as their interface may change. I would prefer to direct the user to the GCP website which has [detailed instructions on requesting quotas](https://cloud.google.com/compute/quotas).

---

> "Name your cluster." Because the project and bucket are pre-specified names, it's not obvious that the cluster is being created here, rather than referencing something else that already exists like for the other two options.

Not sure I understand. The cluster is not being created here, but the user just needs to supply the name for the cluster we will create when they select "create" from the main menu. We just provide some random default names so the user can just click through it.

---

**Comment by ngreenwald** [comment]
I think a link to google's instructions definitely make sense instead of writing our own. I think this is what created the problem in the first place, the instructions were based on an old version of gcloud interface, which ended up being actively unhelpful. 

Precisely. The other times when we had to put a name in, it was the name of something that already existed. So I had to go look and see what the name was, to make sure I put the right one. This step the name is not referencing something already exists, but we're picking what it will be called. But this distinction isn't obvious, so I spent time looking for a cluster name, before I realized we were creating it here. So changing the instructions to "pick a name for your cluster" or something like that to emphasize it's not something they should look.
